### DISCUSSION TITLE
v-data-table release

### DISCUSSION THREAD

**Discussion by Ribeiro-Tiago** [vuetifyjs/vuetify#17318] [Releases]
Hey everyone, is there any ETA / planned date on when v-data-table will leave labs ? Been excitedly waiting to update to vue 3 / vuetify 3 but our product relies a lot of data tables

**Comment by eidottermihi** [comment]
at [it@M](https://github.com/it-at-m) ([platinum sponsor](https://vuetifyjs.com/en/introduction/sponsors-and-backers/#platinum) since april 2023) we are also relying a lot on `v-data-table` in our apps and the current situation does block our efforts to migrate our existing Vue 2 / Vuetify 2 apps to Vue 3 / Vuetify 3.

We would really appreciate an update concerning the current state or a ETA for a stable v-data-table release 🙌

**Reply by rajkumarj-git** [reply]
Can you tell me the vuetify version which can use v-data-table.
I'm struggling in this and i have posted a question stack [https://stackoverflow.com/questions/76804617/vue-3-js-vuetify-component-v-data-table-not-resolving-table-data-object?noredirect=1#comment135403796_76804617](url)
kindly look into my comments I have found a git user used v-data-table in vue 2 and vuetify 2.

**Reply by Ribeiro-Tiago** [reply]
anyone that has v-data-table working is either using it through vuetify labs or using vuetify 2 (with vue 2)

**Comment by ppeelman** [comment]
We as a company also heavily use v-data-table. We would really appreciate an approximate timeline when you envision the component leaving Labs. Thank you for the great library!

**Reply by Ribeiro-Tiago** [reply]
I remember seeing somewhere in the docs that data table was planned for v3.4.0, which according to the [milestones](https://github.com/vuetifyjs/vuetify/milestone/61) is coming out next week

I'm quite skeptical about it buuut that's the closest i've seen to any ETA so far :P

**Comment by eidottermihi** [comment]
With [v3.4.0](https://github.com/vuetifyjs/vuetify/releases/tag/v3.4.0) VDataTable is now `stable` and no longer a "Labs" component 👍

**Comment by Ribeiro-Tiago** [comment]
indeed, closing this discussion as it's no longer necessary
